ext/node/polyfills/internal/crypto/cipher.ts

```diff
@@ -116,6 +118,10 @@ export interface DecipherOCB extends Decipher {
   ): this;
 }
 
+function toU8(input: string | Uint8Array): Uint8Array {
+  return typeof input === "string" ? encoder.encode(input) : input;
```

Comment:
nit: use `core.encode` instead. Ideally we could have seperate fast ops for string and buffer args.